angular/_darkTheme.scss

```diff
@@ -435,4 +435,12 @@
         background-color: rgba(map-get($pxb-black, 50), 0.1);
         color: map-get($pxb-black, 50);
     }
+    .pxb-user-menu-overlay {
+        .pxb-info-list-item-content {
+            color: map-get($foreground, secondary-text);
+            .mat-list-text {
```

Comment:
In dark theme, the ILI subtitle should be text secondary. In light theme it's text primary.